_rules/html-page-lang-matches-default-ucwvc8.md

```diff
@@ -201,7 +205,7 @@ This page has `lang` attribute value of `en` (English), which does not matches t
 
 #### Failed Example 4
 
-This page has `lang` attribute value of `nl` (Dutch), which does not matche the [default language of the page][default page language]. The default language is English because the accessible texts are English, and all other text is in a `p` element with a `lang` attribute value of `nl`.
+This page has `lang` attribute value of `nl` (Dutch), which does not match the [default language of the page][default page language]. The default language is English because the accessible texts are English, and all other text is in a `p` element with a `lang` attribute value of `nl`.
```

Comment:
```suggestion
This page has a `lang` attribute value of `nl` (Dutch), which does not match the [default language of the page][default page language]. The default language is English because the accessible texts are English, and all other text is in a `p` element with a `lang` attribute value of `nl`.
```